client/interfaces.go

```diff
@@ -108,6 +108,14 @@ type ContainerServer interface {
 	MigrateContainerSnapshot(containerName string, name string, container api.ContainerSnapshotPost) (op Operation, err error)
 	DeleteContainerSnapshot(containerName string, name string) (op Operation, err error)
 
+	GetContainerBackupNames(containerName string) (names []string, err error)
+	GetContainerBackups(containername string) (backups []api.ContainerBackup, err error)
+	GetContainerBackup(containerName string, name string) (backup *api.ContainerBackup, ETag string, err error)
+	CreateContainerBackup(containerName string, backup api.ContainerBackupsPost) (op Operation, err error)
+	RenameContainerBackup(containerName string, name string, backup api.ContainerBackupPost) (op Operation, err error)
+	DeleteContainerBackup(containerName string, name string) (op Operation, err error)
+	GetContainerBackupExport(containerName string, name string) (export *api.ContainerBackupExport, ETag string, err error)
```

Comment:
Should be GetContainerBackupFile for consistency with image download

Reply:
I'd also expect it to work in a similar way, pass a BackupFileRequest struct which contains references to the target file, progress handler and canceler and get a BackupFileResponse with the size of the file. This would let us expand the design to transmit multiple files later if needed.